src/backend/distributed/commands/call.c

```diff
@@ -207,3 +225,53 @@ CallFuncExprRemotely(CallStmt *callStmt, DistObjectCacheEntry *procedure,
 
 	return true;
 }
+
+
+#if PG_VERSION_NUM >= PG_VERSION_14
+
+/*
+ * FunctionHasOutOnlyParameter is a helper function which takes
+ * a function Oid and returns true if the input function has at least
+ * one OUT parameter.
+ */
+static bool
+FunctionHasOutOnlyParameter(Oid functionOid)
```

Comment:
Minor: I try to use `id` instead of `Oid` in variable names since it seems more natural to me. Just sharing as a thought